components/ide/jetbrains/gateway-plugin/src/main/kotlin/io/gitpod/jetbrains/gateway/GitpodWorkspacesView.kt

```diff
@@ -260,6 +277,22 @@ class GitpodWorkspacesView(
                                     }.rowComment("<a href='${info.workspace.context.normalizedContextURL}'>${info.workspace.context.normalizedContextURL}</a>")
                                 }
                                 label("").resizableColumn().horizontalAlign(HorizontalAlign.FILL)
+                                panel {
+                                    val repo = info.latestInstance.status.repo
+                                    val changes = repo?.let {
+                                        it.totalUncommitedFiles + it.totalUntrackedFiles + it.totalUnpushedCommits
+                                    } ?: 0
+                                    row {
+                                        label(info.workspace.context.ref)
+                                    }.rowComment(
+                                        if (changes > 0) {
+                                            "<b>$changes Changes</b>"
+                                        } else {
+                                            "No Changes"
+                                        }
```

Comment:
Could you also check for the singular change case here?

On webapp:
<img width="102" alt="image" src="https://user-images.githubusercontent.com/418083/160440605-a861de06-389f-49df-b15f-4fd345f79900.png">

On gateway:
<img width="94" alt="image" src="https://user-images.githubusercontent.com/418083/160440760-735b1257-7bf6-4892-8bd0-f60a1f8c34bb.png">

Reply:
Thanks @felladrin for your comment!

Due to the limitation of JetBrains Plugins's UI framework (Java-GUI based), the user experience (also the L&F of UI components) on GW would be a bit different comparing with on WebApp. 

I would make a try to add a dropdown button to display the detailed changed files, which might looks similar to the following "Item 1" dropDownLink, is that acceptable?

<img width="810" alt="1" src="https://user-images.githubusercontent.com/3115235/160503415-3482b3ca-da7e-4cff-8286-f6c61d2cdd76.png">

Reply:
Sorry I misunderstood the comment 😅
I just fixed the hardcoded plural changes. PTAL @felladrin

Reply:
Thanks for the update! 
It's perfect now!